instrumentation/activity/src/test/java/io/opentelemetry/android/instrumentation/activity/VisibleScreenTrackerTest.java

```diff
@@ -22,17 +22,17 @@ void activityLifecycle() {
         VisibleScreenTracker visibleScreenTracker = new VisibleScreenTracker();
         Activity activity = mock(Activity.class);
 
-        assertEquals("unknown", visibleScreenTracker.getCurrentlyVisibleScreen());
+        Assertions.assertEquals("unknown", visibleScreenTracker.getCurrentlyVisibleScreen());
```

Comment:
I've seen a lot of these changes which seem to make the PR bigger than it should be, is it needed to change the imports for these tests?

Reply:
No, this was unintentional. I'll remove them. Thanks!